packages/widget-v2/src/pages/SwapPage/useSetMaxAmount.ts

```diff
@@ -21,13 +21,18 @@ export const useGasFeeTokenAmount = () => {
     chainId: sourceAsset?.chainID,
   });
 
+  const ethDetails = useGetAssetDetails({
+    assetDenom: "ethereum-native",
+    chainId: "1"
```

Comment:
you need to specify the denom for different chainId if you do it like this. It's easier to not do this because in evm it's not always eth for the fee asset